WalletWasabi.Backend/wwwroot/css/style.css

```diff
@@ -491,6 +491,12 @@ section {
     margin-right: 17px;
 }
 
+.icon-colored-clearnet {
+    width: 25px;
```

Comment:
```suggestion
    width: 17px;
```

Reply:
The onion icon looks great with those settings, but the clearnet icon is not.

Reply:
I prefer it with `25 px`, it's more clear and almost the same size regardless of px value.

Reply:
Ok, LGTM